de-lib/src/main/java/org/iplantc/de/theme/base/client/desktop/Desktop.gss

```diff
@@ -168,7 +172,7 @@
     gwt-sprite: "dataBtn";
     height: WIN_BTN_HEIGHT !important;
     width: WIN_BTN_WIDTH !important;
-    margin-top: WIN_BTN_MARGIN_TOP;
```

Comment:
Yeah, making the banner bigger meant that I had to push the Data icon down further, otherwise it was only a few pixels below the banner which looked crowded.  The other icons then are spaced evenly away from the Data icon.